src/pages/components/data-table/code.mdx

```diff
@@ -1,32 +1,91 @@
 ---
 title: Data table
 description: The data table component allows for the flexible display and sorting of information.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
-import { rowData, headerData } from '../../../data/components/data-table.js';
-
-## Default
+import { rowData, headerData } from "../../../data/components/data-table.js";
 
 <ComponentDemo
```

Comment:
Theme bug: Zebra stripes aren't rendering correctly

![image](https://user-images.githubusercontent.com/11670886/78406224-d1fa3980-75c7-11ea-95e2-18127f354da6.png)

Reply:
Also can we add the batch action toolbar to the selection variant. I know people will ask about it.